data_steward/tools/import_rdr_dataset.py

```diff
@@ -0,0 +1,231 @@
+
+        schema_list = client.get_table_schema(table, schema)
+        destination_table = bigquery.Table(destination_table_id,
+                                           schema=schema_list)
+
+        for schema_item in schema_list:
+            if 'person_id' in schema_item.name and table.lower(
+            ) != 'pid_rid_mapping':
+                destination_table.clustering_fields = 'person_id'
+                destination_table.time_partitioning = bigquery.table.TimePartitioning(
+                    type_='DAY')
+
+        LOGGER.info(
+            f'Loading `{source_table_id}` into `{destination_table_id}`')
+
+        try:
+            LOGGER.info(f'Get table `{source_table_id}` in RDR')
+            client.get_table(source_table_id)
+
+            LOGGER.info(f'Creating empty CDM table, `{table}`')
+            destination_table = client.create_table(
+                destination_table)  # Make an API request.
+
+            LOGGER.info(
+                f'Copying source table `{source_table_id}` to destination table `{destination_table_id}`'
+            )
+
+            sc_list = []
+            for item in schema_list:
+                field_cast = f'CAST({item.name} AS {BIGQUERY_DATA_TYPES[item.field_type.lower()]}) AS {item.name}'
+                sc_list.append(field_cast)
+
+            fields_name_str = ',\n'.join(sc_list)
+
+            # copy contents from source dataset to destination dataset
+            sql = (f'SELECT {fields_name_str} ' f'FROM `{source_table_id}`')
+
+            job_config = bigquery.job.QueryJobConfig(
+                write_disposition=bigquery.job.WriteDisposition.WRITE_EMPTY,
+                priority=bigquery.job.QueryPriority.BATCH,
+                destination=destination_table,
+                labels={
+                    'table_name': table.lower(),
+                    'copy_from': rdr_source_dataset.split('.')[1].lower(),
+                    'copy_to': destination_dataset.lower()
+                })
+            job_id = (f'schemaed_copy_{table}_'
+                      f'{datetime.now().strftime("%Y%m%d_%H%M%S")}')
+            job = client.query(sql, job_config=job_config, job_id=job_id)
+            job.result()  # Wait for the job to complete.
+        except NotFound:
+            LOGGER.info(
+                f'{table} not provided by RDR team.  Creating empty table '
+                f'in dataset: `{destination_dataset}`')
+
+            LOGGER.info(f'Creating empty CDM table, `{table}`')
+            destination_table = client.create_table(destination_table)
+            LOGGER.info(f'Created empty table `{destination_table.table_id}`')
+        else:
+            destination_table = client.get_table(
+                destination_table_id)  # Make an API request.
+        LOGGER.info(f'Loaded {destination_table.num_rows} rows into '
+                    f'`{destination_table.table_id}`.')
+
+    LOGGER.info(f"Finished RDR table LOAD from dataset {rdr_source_dataset}")
```

Comment:
We have this exact function in import_rdr_omop.py, you can get rid of it here and just import the function from that script.

Reply:
are you referring to `copy_vocab_tables` ?

Reply:
Yes